metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/NetworkFunctions.java

```diff
@@ -182,8 +186,8 @@ public Object apply(List<Object> objects) {
   public static class URLToPath extends BaseStellarFunction {
```

Comment:
Can you add testing around this?  In particular, make sure that fragments are handled correctly (`URI` has a separate `getFragment()`, so I assume that they are not returned by `getPath()`.  I could be wrong about this, though, I haven't run a test on it.).  If this is the case, we may also want/need a `TO_FRAGMENT` function that handles this appropriately, although I'm fine with that being a separate conversation/jira.